openwisp_ipam/tests.py

```diff
@@ -140,3 +141,32 @@ class TestForms(BaseTestForms, TestCase):
 class TestModel(BaseTestModel, CreateModelsMixin, TestCase):
     ipaddress_model = swapper.load_model('openwisp_ipam', 'IPAddress')
     subnet_model = swapper.load_model('openwisp_ipam', 'Subnet')
+
+
+class TestUserAdmin(TestCase):
+    user_model = get_user_model()
+
+    def test_create_user(self):
+        self.user_model.objects.create_superuser(
+            username='admin',
+            password='qwertyuiop',
+            email='',
+        )
+        test_client = Client()
```

Comment:
why are you doing this? The test client instance is available by default in `self.client`

Reply:
I didn't know, I'll get rid of this and use `self.client`